### ISSUE TITLE
[Feature] Animate damage numbers on assignment, not on roll.

### ISSUE THREAD

**Issue by maplealmond** [ChasarooniZ/pf2e-rpg-numbers#16]
Damage numbers are neat, but if you're a class which relies heavily vulnerabilities, or even if you just want to know what extra oomph, the numbers don't show.

Showing the total damage that gets applied (e.g. when the message showing that damage has been applied) would be a nice option.  It might require a total consolidated value instead of a bunch of smaller numbers, but this is an acceptable tradeoff to see the final value.

The shake on the damage application is also fine as the GM tends to describe the hit after the damage numbers are properly rolled (and no shake when hitting a creature who is immune makes for a fun no-sell)

**Comment by ChasarooniZ** [comment]
Yeah I think that feature should be easy to implement, I'll look into it

**Comment by ChasarooniZ** [comment]
Finished with newest release

**Comment by maplealmond** [comment]
Thank you so much for this.

I think there might be an issue with the release.  I do not see the setting to actually turn on the apply.

After further investigation, while my module.json confirms this is 11.8.3, the scripts/module.js haes no mention of dmg-on-apply-or-roll - confirmed this is missing from module.zip in the latest release.

**Comment by ChasarooniZ** [comment]
11.8.4 should have it
